dspace-server-webapp/src/test/java/org/dspace/app/rest/signposting/controller/LinksetRestControllerIT.java

```diff
@@ -369,20 +458,50 @@ public void findOneItemLsetLinksets() throws Exception {
+        String mimeType = "application/vnd.datacite.datacite+xml";
+        String siteAsRelation = "<" + url + "/handle/" + item.getHandle() + "> ; rel=\"cite-as\" ; anchor=\"" +
                 url + "/entities/publication/" + item.getID() + "\" ,";
         String itemRelation = "<" + url + "/bitstreams/" + bitstream1.getID() +
                 "/download> ; rel=\"item\" ; " + "type=\"text/plain\" ; anchor=\"" + url + "/entities/publication/" +
                 item.getID() + "\" ,";
-        String typeRelation = "<https://schema.org/AboutPage> ; rel=\"type\" ; type=\"text/html\" ; anchor=\"" +
-                url + "/entities/publication/" +
-                item.getID() + "\" ,";
+        String typeRelation = "<https://schema.org/AboutPage> ; rel=\"type\" ; anchor=\"" +
+                url + "/entities/publication/" + item.getID() + "\" ,";
+        String linksetRelation = "<" + url + "/" + signpostingUrl + "/linksets/" + item.getID().toString() +
+                "> ; rel=\"linkset\" ; type=\"application/linkset\" ;" +
+                " anchor=\"" + url + "/entities/publication/" + item.getID() + "\" ,";
+        String jsonLinksetRelation = "<" + url + "/" + signpostingUrl + "/linksets/" + item.getID().toString() +
+                "/json> ; rel=\"linkset\" ; type=\"application/linkset+json\" ;" +
+                " anchor=\"" + url + "/entities/publication/" + item.getID() + "\" ,";
+        String describedByRelation = "<" + url + "/" + signpostingUrl + "/describedby/" + item.getID() +
+                "> ; rel=\"describedby\" ;" + " type=\"" + mimeType + "\" ; anchor=\"" + url +
+                "/entities/publication/" + item.getID() + "\" ,";
+
+        String bitstreamCollectionLink = "<" + url + "/entities/publication/" + item.getID() + "> ;" +
+                " rel=\"collection\" ; type=\"text/html\" ; anchor=\"" + url + "/bitstreams/"
+                + bitstream1.getID() + "/download\"";
+        String bitstreamLinksetLink = "<" + url + "/" + signpostingUrl + "/linksets/" + item.getID() + "> ; " +
+                "rel=\"linkset\" ; type=\"application/linkset\" ; " +
+                "anchor=\"" + url + "/bitstreams/" + bitstream1.getID() + "/download\"";
+        String bitstreamLinksetJsonLink = "<" + url + "/" + signpostingUrl + "/linksets/" + item.getID() + "/json> ; " +
+                "rel=\"linkset\" ; type=\"application/linkset+json\" ; " +
+                "anchor=\"" + url + "/bitstreams/" + bitstream1.getID() + "/download\"";
+
+        String describesMetadataLink = "<" + url + "/entities/publication/" + item.getID() + "> ; " +
+                "rel=\"describes\" ; type=\"text/html\" ; " +
+                "anchor=\"" + url + "/" + signpostingUrl + "/describedby/" + item.getID() + "\"";
 
-        getClient().perform(get("/signposting/linksets/" + item.getID())
-                        .header("Accept", "application/linkset"))
+        getClient().perform(get("/signposting/linksets/" + item.getID()))
```

Comment:
please add an expectation about the content-type response header. It must be `application/linkset` (it is not required that the client ask for this format but the server must inform the client about the type of response)

Reply:
fixed